CenterForOpenScience/ember-osf-web
# Feature/no more filemanager

## Purpose

Increase code coverage by removing unused, untested code.

## Summary of Changes

1. Remove file-manager service
2. Remove file-manager tests (all skipped)
3. Remove unused reference to file manager

## QA Notes

This code should be entirely unused, but there was a note that it was attempting to fix a url reloading race condition when creating or modifying a file. So try to do quick operations on a file (rename then rename again, rename then go to detail page, or things like that).

## Ticket

N/A

# Reviewer Checklist
